tests/sentry/search/events/test_fields.py

```diff
@@ -335,8 +335,8 @@ def test_field_alias_expansion(self):
             "issue.id",
             ["coalesce", ["user.email", "user.username", "user.ip"], "user.display"],
             "message",
-            "timestamp.to_hour",
-            "timestamp.to_day",
+            ["toStartOfHour", ["timestamp"], "timestamp.to_hour"],
+            ["toStartOfDay", ["timestamp"], "timestamp.to_day"],
```

Comment:
To be consistent with `user.display`, field aliases for these timestamp fields should also group by the full expression.